send_multiple_messages.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import sys
+import threading
+
+from client import send_data
+
+
+def send_messages(server_ip, server_port, messages):
+    """
+    Send messages in parallel to server in address (server_port, server_ip)
+    """
+    for message in messages:
+        threading.Thread(target=send_data, args=(server_ip, server_port, message)).start()
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Send multiple messages to server in parallel.')
+    parser.add_argument('server_ip', type=str,
+                        help='the server\'s ip')
+    parser.add_argument('server_port', type=int,
+                        help='the server\'s port')
+    parser.add_argument('messages', type=str, nargs=argparse.REMAINDER,
```

Comment:
Are you sure that the type here is `str`? It seems like you are using this as a list of strings rather than a string itself.

Reply:
AFAIK `type` is the type of each *individual* argument, and `nargs` is the number of arguments. I found this example online: https://gist.github.com/livibetter/2581659